packages/device-connector/src/devices/deployment.ts

```diff
@@ -64,6 +64,7 @@ export const deployBinary = async (deployData: DeploymentData): Promise<void> =>
   }
 
   const artifactPath = await downloadArtifact(artifactUri)
+  logger.debug(`Deployment: ${deployData.id} - Downloaded artifact from ${artifactUri} to ${Path.basename(artifactPath)}`)
```

Comment:
```suggestion
  logger.debug(`Deployment: ${deployData.id} - Downloaded artifact from ${artifactUri} to ${basename(artifactPath)}`)
```

`import * as Path from 'path'` is an unconventional import

Reply:
changed in 37f119b16a9a2ece04526ab67f65e60611370b7b